### ISSUE TITLE
Proper support and integration for Chat registry

### ISSUE THREAD

**Issue by justEli** [PaperMC/Velocity#749]
Messages sent to chat by (my) plugins in Velocity, using kyori, get prefixed with '<> ', eversince 1.19. The following code was used:

```java
TextComponent.Builder component =
  Component.text()
    .append(Component.text(part1))
    .append(Component.text(part2));

this.proxy.getAllPlayers().forEach(player -> player.sendMessage(component));
```
![image](https://user-images.githubusercontent.com/8517465/173445405-065dd65e-a5f1-4475-81de-13136d951c8c.png)
This did not happen pre-1.19. It also does not happen for any message sent from a Paper server.

Velocity 3.1.2-SNAPSHOT (git-74edac96-b153)

**Comment by electronicboy** [comment]
if viaversion, update to latest dev build

**Comment by justEli** [comment]
ViaVersion is not installed
NuVotifier, LiteBans and LuckPerms are

**Comment by Xernium** [comment]
The reason this happens is because the current version of Velocity can‘t be aware of the ID from the chat-registry it should be using. This will be fixed in the near future.

**Comment by justEli** [comment]
thank you

**Comment by justEli** [comment]
any updates on this?

**Comment by electronicboy** [comment]
Work is currently on a branch, the <> should of been fixed this morning, however

**Comment by davidmayr** [comment]
> Work is currently on a branch, the <> should of been fixed this morning, however

Just updated to the Branch dev/chat-registry (compiled it myself) and I still get them.
On the Backend Servers I run the current Release of ViaVersion and not a Dev-Build though. Maybe that's the issue?

I'm just not Upgrading because of this issue: ViaVersion/ViaVersion#2945

**Comment by electronicboy** [comment]
Yea, via is going to induce that issue as they send an unexpected registry which causes an id mismatch, that should be fixed in their dev builds

**Comment by davidmayr** [comment]
> Yea, via is going to induce that issue as they send an unexpected registry which causes an id mismatch, that should be fixed in their dev builds

But it's not. Using the newest ViaVersion Dev-Build(https://ci.viaversion.com/job/ViaVersion-DEV/712/) you will receive messages that were sent before you joined a server with the <> in front of the text but not any messages after that. This build makes all Velocity messages disappear. (As mentioned in the issue I linked)

**Comment by electronicboy** [comment]
<img width="856" alt="image" src="https://user-images.githubusercontent.com/1228900/175165410-f80980b8-6b00-42bf-9294-37c3e78a113a.png">
I cannot reproduce this issue on the main dev builds, the chat registry stuff is still a work in progress and missing some steps to handle some cases

DO note that when I said fixed this morning, I meant my commit in the snapshot branch, not the chat registry branch, I've not  handled the servers chat packet yet to actually use the registry, and not sure if I committed 100% of my stuff on the registry branch yet

**Comment by justEli** [comment]
it is fixed for my server, thanks

**Comment by davidmayr** [comment]
Yeah, that fixed it. And after updating ViaVersion to its dev build, it's working without any problems.
